app/controllers/api/v1/collections_controller.rb

```diff
@@ -39,7 +39,7 @@ def check_default_subject(collection)
   def filter_by_project_ids
     if ids_string = (params.delete(:project_ids) || params.delete(:project_id)).try(:split, ',')
       project_ids = ids_string.split(",")
-      @controlled_resources = controlled_resources.where.overlap(project_ids: project_ids)
+      @controlled_resources = controlled_resources.joins(:projects).where(projects: {id: project_ids})
```

Comment:
can't this just be `controlled_resources.projects.where(id: project_ids)` ?

Reply:
That would return projects, but we need to return collections?